### PR TITLE
 [OBSDEF-9089] Added events and issue rules to report SNMP traps 

### PR DESCRIPTION
**Pull request by vgupta-mickey**
## Purpose
[OBSDEF-9089](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-9089)
Added a new SNMP notifier as  a part of issue and event rules to report TRAPs to the SNMP server.
- All issue rules will add SNMP notifier to report traps on creating/updating/clearing any issue for all applications
- All Critical, Warning, Error events will be reported to SNMP server for ObjectStore and Objectscale-operator

## PR checklist
- [ x] make test passed
- [ x] make build passed
- [x ] helm install <chart> passed
- [x ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
This is simple rules. I tested with sample-test application.
It works.

### CODE REVIEW

File: ecs-cluster/alerts/issues/1006_BUCKET_HARD_QUOTA_EXCEEDED_issue.yaml [hunk 1]
```diff
@@ -6,4 +6,6 @@
         # ERROR
         - label: SymptomID
           value: OBJST-1006
+  notifiers:
```
Comment (vazhem) [inline ecs-cluster/alerts/issues/1006_BUCKET_HARD_QUOTA_EXCEEDED_issue.yaml:9]: Ideally I would prefer to not specify every possible notifier for every issue rule.
Issue should have an attribute like channel: ESE / Customer / All.
This way when you would add other notifiers like smtp, syslog, there will be no need to specify all possible supported notifiers in every issue definition...

Comment (vgupta-mickey) [reply ecs-cluster/alerts/issues/1006_BUCKET_HARD_QUOTA_EXCEEDED_issue.yaml:9]: I think we can discuss on this but not on this PR. It needs KAHM infrastructure code change and I don’t think that we have time to change that now.

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]
Please run custom-charts ci jenkins job.

File: supportassist/templates/supportassist-app-configmap.yaml [hunk 2]
```diff
@@ -24,6 +24,8 @@ data:
           - matchon:
             - label: SymptomID
               value: DECKS-SA-1023
+         notifiers:
+          - {{.Values.product}}-snmp-notifiers
```
Comment (eddiepavkovic) [inline supportassist/templates/supportassist-app-configmap.yaml:28]: typo
```suggestion
          - {{.Values.product}}-snmp-notifier
```

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]
typo on notifier.

Also please indicate what testing was done... if possible show objecstore, license or objectscale-manager snmp traps being received.

**Review by eddiepavkovic — APPROVED** [review]
nice.  lgmt

**Review by ben-schumacher — APPROVED** [review]

**Pull request merged**